packages/builder/friendly-errors-webpack-plugin/README.md

```diff
@@ -0,0 +1,162 @@
+# Friendly-errors-webpack-plugin
+
```

Comment:
- We should add a new credits to the root README.md
- We should add copyright at the top of every source files. same as https://github.com/web-infra-dev/modern.js/blob/10572f3a89a7a64daf5e4f13cbfef09902669033/packages/builder/builder-shared/src/loaders/css-modules-typescript-loader.ts#L1